MANUAL.md

```diff
@@ -452,35 +453,89 @@ After you [have set the preferred filter parameters for incidents](#decide-which
 ## Customize alarm notifications in Argus
 Choose **when**, **where** and **what** alarm notifications you want to receive by creating, editing and deleting _notification profiles_.
 
+  <img src="public/screenshots/manual/Screenshot 2023-04-26 at 14.16.54.png"/>
+
+
+### About components of notification profiles
+1. **Timeslot** allows you to customize **when** you want to receive the alarm notifications. You can choose one timeslot per notification profile. Timeslots are reusable across multiple notification profiles.
+2. **Filter** allows you to customize **what** alarms (incidents) you want to receive the notifications about. You can choose multiple filters per notification profile. Filters are reusable across multiple notification profiles.
+3. **Destination** allows you to customize **where** you want to receive the alarm notifications. You can choose multiple destinations per notification profile. Destinations are reusable across multiple notification profiles. Destinations may be of [different media types](#about-the-available-notification-media).
+
+
+### About the available notification media
+The notification media that are available in Argus by default are:
+- SMS
+- Email
+
+If you wish to receive notifications to other media, read about configurable media types [here](https://argus-server.readthedocs.io/en/latest/notifications.html#other-notification-plugins).
```

Comment:
"here"-links are evil. Try replacing with:

`media types  in the [docs for notifications]`

or something. Think what would make a good bookmark text. Especially important with markdown since you cannot add `title` and other html attributes.

(This does not mean you should hunt down every here-link in the docs and fix them in this PR. Clean 'em away the next time you need to do something with the file anyway.)